test/runFIT.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash -ex
+
+############################################
+# Global variable
+############################################
+RACKHD_TEST_DIR="$( cd -P "$( dirname "$0" )" && pwd )"
+
+#############################################
+#
+# Usage
+############################################
+Usage(){
+    set +x
+    echo "Function: This script is used to set up environment for FIT and run FIT."
+    echo "Usage: $0 [OPTIONS]"
+    echo "  OPTIONS:"
+    echo "    Mandatory options:"
+    echo "      -w, --WORKSPACE: The directory of workspace( where the code will be cloned to and staging folder), it's required"
```

Comment:
if put runFIT.sh as a common script,  the WORKSPACE is not mandatory .. because what you use WORKSPACE is just to copy the xunit reports.